openrr-teleop/src/control_node.rs

```diff
@@ -5,7 +5,7 @@ use auto_impl::auto_impl;
 #[async_trait]
 #[auto_impl(Box)]
 pub trait ControlNode: Send + Sync {
-    fn set_event(&mut self, event: GamepadEvent);
+    async fn set_event(&mut self, event: GamepadEvent);
```

Comment:
Notice that `ControlNode::set_event` also changed to async because some implementers are calling `Speaker::speak`.